ui/next.config.js

```diff
@@ -2,4 +2,55 @@ const withTM = require('next-transpile-modules')(['domain-logic'])
 
 module.exports = withTM({
   experimental: { esmExternals: true },
+  async headers() {
+    return [
+      {
+        source: '/(.*)',
+        headers: securityHeaders,
+      },
+    ]
+  },
 })
+
+// The declaration below will allow Youtube Videos and Google Fonts
+const ContentSecurityPolicy = `
+  default-src 'self';
+  script-src 'self' 'unsafe-eval' 'unsafe-inline' *.youtube.com;
+  child-src *.youtube.com;
+  style-src 'self' 'unsafe-inline' *.googleapis.com;
```

Comment:
having unsafe-inline for styles is potentially useless as well

Reply:
Now we'll only have `unsafe-inline` in development

it seems that Nextjs only need those in development mode \o/